zerver/webhooks/github/view.py

```diff
@@ -874,6 +874,12 @@ def get_zulip_event_name(
             return "pull_request_auto_merge"
         if action in IGNORED_PULL_REQUEST_ACTIONS:
             return None
+    # Github sends 2 events for reviews with comments. See #26145
+    elif header_event == "pull_request_review":
+        action = payload["action"].tame(check_string)
+        if action == "edited":
+            return None
```

Comment:
This would lead to every PR review edit event being ignored; we only want to ignore the extra edited event that comes after creating a PR review. See the issue description for more details.